### DISCUSSION TITLE
Gallery of Jekyll Themes for GitHub Pages

### DISCUSSION THREAD

**Discussion by Garcia-INPE** [cu-esiil-edu/2024-data-short-course#14] [Show and tell]
For those who are interested in choosing the right Jekyll's theme for the GitHub Page you are building I made a gallery of them. **`Have fun and impress!`**
![Jekyll Themes for GitHub Pages gallery - Garcia-INPE](https://github.com/cu-esiil-edu/2024-data-short-course/assets/99909390/278edbff-4dba-43ab-b2a1-a1ef55f17327)

**Comment by nquarder** [comment]
Very nice @Garcia-INPE and thank you for putting together this helpful resource! Much appreciated.

**Comment by oye-bobs** [comment]
Thank you @Garcia-INPE , very nice.
